src/scripts/ci/publish.ts

```diff
@@ -293,18 +293,18 @@ export function getPublishOrder(packages: Packages): string[][] {
 }
 
 /**
- * Takes the max alpha version + 1
- * For now supporting 2.0.0-alpha.X
+ * Takes the max dev version + 1
+ * For now supporting 2.0.0-dev.X
  * @param packages Local package definitions
  */
-async function getNewAlphaVersion(packages: Packages): Promise<string> {
+async function getNewDevVersion(packages: Packages): Promise<string> {
   const before = Date.now()
-  console.log('\nCalculating new alpha version...')
-  const versions = await getAllVersions(packages, 'alpha')
-  const alphaVersions = getAlphaVersionIncrements(versions)
-  const maxAlpha = Math.max(...alphaVersions)
+  console.log('\nCalculating new dev version...')
+  const versions = await getAllVersions(packages, 'dev')
+  const devVersions = getDevVersionIncrements(versions)
+  const maxDev = Math.max(...devVersions, 0)
 
-  const version = `2.0.0-alpha.${maxAlpha + 1}`
+  const version = `2.0.0-dev.0${maxDev + 1}`
```

Comment:
looks like this is `2.0.0-dev.01` to me?
Why not 
```suggestion
  const version = `2.0.0-dev.${maxDev + 1}`
```